R/hiper_glm.R

```diff
@@ -1,31 +1,104 @@
+
+####################### Helper functions ######################
+
+########## For cholesky ########
+## Find linear regression coefficients estimates via MLE,
+## with cholesky decomposition while solving matrices
+chol_finder <- function(design, outcome){
```

Comment:
Calling this the pseudoinverse finder instead of cholesky might be more appropriate, since it's the method being used to compute the MLE. You could _also_ define a chol function as the method used to compute the pseudoinverse, if you wanted to further add a layer of abstraction.